sell/sell-api.yaml

```
@@ -3125,40 +2857,169 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ExtendedQuote"
-    delete:
+  /public/invoices/{id}:
+    get:
       tags:
-        - Quotes
-      summary: Delete a quote
-      description: Use this endpoint to delete an obsolete quote.
-      operationId: deleteQuote
+        - Invoices
+      summary: Retrieve an invoice
```

```suggestion
      summary: Generate a public invoice
```
I'm afraid we have to rename the summary otherwise it will show up a duplicate entry in API docs (we already have another 'Retrieve an invoice' with /invoices/{id})

---

Done